cms-cat.spec

```diff
@@ -24,19 +24,6 @@ mv * %{i}/cat
 cd ${RPM_INSTALL_PREFIX}/%{pkgrel}
 mkdir -p ${RPM_INSTALL_PREFIX}/cat ${RPM_INSTALL_PREFIX}/etc/%{pkgname}
 
-#Check if a newer revision is already installed
-if [ -f ${RPM_INSTALL_PREFIX}/etc/%{pkgname}/version ] ; then
-  oldrev=$(cat ${RPM_INSTALL_PREFIX}/etc/%{pkgname}/version )
-  if [ ${oldrev} -ge %{fakerevision} ] ; then
-    exit 0
-  fi
-fi
```

Comment:
@clelange , you need to keep this above block otherwise an old version of cms-cat (e.g. when it will be rebuild/installed for an old release cycle) will override the new version files

Reply:
OK, done. Sorry, I don't know how to test this myself, so I'm relying on you here.